homeassistant/components/nuki/lock.py

```diff
@@ -96,7 +106,7 @@ def is_locked(self):
     def device_state_attributes(self):
         """Return the device specific state attributes."""
         data = {
-            ATTR_BATTERY_CRITICAL: self._battery_critical,
+            ATTR_BATTERY_LEVEL: 10 if self._battery_critical else 100,
```

Comment:
We shouldn't make up values. The user doesn't know what's going on.

It's better to extract it as a binary sensor and use the `battery` device class. https://developers.home-assistant.io/docs/en/entity_binary_sensor.html